packages/dashboard/src/components/pages/platforms/components/instruments/instruments.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { TStringTranslation } from 'Types'
+
+const Instruments: React.FC<IInstrumentsType> = ({ title, active_symbols = [] }) => {
+    return (
+        <div className='dw-instruments'>
+            <Text className='dw-instruments__title' size='m' weight='bold'>
+                {title}
+            </Text>
+            {active_symbols.map((market, idx) => (
+                <React.Fragment key={idx}>
+                    <div className='dw-instruments__market'>
+                        <Icon className='dw-instruments__market-icon' icon={market.icon} width='48' height='48' />
+                        <div className='dw-instruments__market-title'>
+                            <Text size='sm' weight='bold'>
+                                {market.title}
+                            </Text>
+                            {market.description && <Text>{market.description}</Text>}
+                        </div>
+                    </div>
+                    {market.submarkets.map((submarket, idy) => (
+                        <div key={idy} className='dw-instruments__submarket'>
```

Comment:
Same here, use the `submarket` name as a key rather than `idx`

Reply:
same here, will rename it to submarket_index

Reply:
See https://github.com/binary-com/deriv-app/pull/2476#discussion_r556974153